docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/creating-individual-params.md

```diff
@@ -0,0 +1,95 @@
+
+- You must replace the `sample_sensor_kit` with the one you forked when creating the Autoware repository page.
+
+```diff
+autoware_individual_params/individual_params/config/<VEHICLE-ID>/
+-   ├─ sample_sensor_kit/
++   └─ VEHICLE-ID_sensor_kit/
+```
+
+### Folder Structure
+
+The folder structure of forked `individual_params` should be like this:
+
+```diff
+individual_params/
+  └─ config/
+       └─ VEHICLE-ID/
+            └─ VEHICLE-ID_sensor_kit/
+                 ├─ imu_corrector.param.yaml
+                 ├─ sensor_kit_calibration.yaml
+                 └─ sensors_calibration.yaml
+```
+
+## imu_corrector.param.yaml
+
+A file used by imu_corrector.
+
+The default [imu_corrector.param.yaml](https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/imu_corrector/config/imu_corrector.param.yaml) included in [imu_corrector_package](https://github.com/autowarefoundation/autoware.universe/tree/main/sensing/imu_corrector),
+but it will be overridden by [autoware_individual_param](https://github.com/autowarefoundation/autoware_individual_params/tree/main/individual_params).
+The `imu_corrector_param.yaml` created for `sample_sensor_kit` is given below:
+
+```yaml
+/**:
+  ros__parameters:
+    angular_velocity_stddev_xx: 0.00339
+    angular_velocity_stddev_yy: 0.00339
+    angular_velocity_stddev_zz: 0.00339
+    angular_velocity_offset_x: -0.005799
+    angular_velocity_offset_y: -0.007148
+    angular_velocity_offset_z: -0.001499
+```
+
+`imu_corrector_param.yaml` is calculated with [deviation_estimation_tools](https://github.com/tier4/CalibrationTools/blob/tier4/universe/localization/deviation_estimation_tools/ReadMe.md).
+
+The tutorial_vehicle's [imu_corrector.param.yaml](https://github.com/leo-drive/tutorial_vehicle_individual_params/blob/main/individual_params/config/tutorial_vehicle/tutorial_vehicle_sensor_kit/imu_corrector.param.yaml) file is given below:
+
+??? note "imu_corrector_param.yaml for tutorial_vehicle_sensor_kit"
+
+    ```yaml
+    /**:
+      ros__parameters:
+        angular_velocity_offset_x: -0.00017
+        angular_velocity_offset_y: 0.00206
+        angular_velocity_offset_z: -0.00006
+        angular_velocity_stddev_xx: 0.05779
+        angular_velocity_stddev_yy: 0.05779
+        angular_velocity_stddev_zz: 0.05779
+    ```
+
+## sensor_kit_calibration.yaml
+
+- How it is created is explained in [Creating sensor kit description page](./creating-sensor-description.md).
+  `sensor_kit_calibration.yaml` will be same file which included in sensor_kit_description package. After the calibration
+  process, the calibration values will be changed.
+
+## sensors_calibration.yaml
+
+- How it is created is explained in [Creating sensor kit description page](./creating-sensor-description.md).
+  `sensor_kit_calibration.yaml` will be same file which included in sensor_kit_description package. After the calibration
+  process, the calibration values will be changed.
```

Comment:
THis is not `sensor_kit_calibration.yaml` There will not be a base_link - lidar calibration process. So we can update it like this:

```suggestion
`sensors_calibration.yaml` will be the same file included in the `sensor_kit_description` package. After obtaining the main sensor frame (sensor_kit_base_link) positions related to the base_link (CAD data can be used for this purpose), these values will be changed.
```